CHANGELOG.md

```diff
@@ -8,50 +8,64 @@
 
 ### Major Changes
 
-* `aframe-core` merged with `aframe`. `window.AFRAME` exposes what `aframe-core` was exposing previously (e.g., `AFRAME.registerComponent` vs.  `AFRAME.aframeCore.registerComponent`). (#368)
-* `<a-assets>` must be declared within `<a-scene>`. (#910)
-* `<a-entity>.object3D` is now a `THREE.Group`. Use `<a-entity>.setObject3D` API to add new 3D objects from components. (#847)
-* Bumped three.js to r74 stable. (#1006)
-* npm points to a prebuilt bundle of `dist/aframe.js`.
-* Scene `canvas` elements are now appended to the scene by default rather than to the document body. The scene can specify the canvas to render to. (c0aa360)
-* Primitives such as `<a-sphere>` directly extend `<a-entity>` rather than template them. (#883)
-* `<a-template>` and HTML imports logic have been removed. (#883)
-* `<a-camera>` no longer creates a cursor. Use `<a-cursor>` instead. (#883)
+- `aframe-core` merged with `aframe`. `window.AFRAME` exposes what `aframe-core` was exposing previously (e.g., `AFRAME.registerComponent` vs.  `AFRAME.aframeCore.registerComponent`). (#368)
+- `<a-assets>` must be declared within `<a-scene>`. (#910)
+- `<a-entity>.object3D` is now a `THREE.Group`. Use `<a-entity>.setObject3D` API to add new 3D objects from components. (#847)
+- Bumped three.js to r74 stable. (#1006)
+- npm points to a prebuilt bundle of `dist/aframe.js`.
+- Scene `canvas` elements are now appended to the scene by default rather than to the document body. The scene can specify the canvas to render to. (c0aa360)
+- Primitives such as `<a-sphere>` directly extend `<a-entity>` rather than template them. They can be registered with `AFRAME.registerPrimitive`. (#883)
+- `<a-template>` and HTML imports logic have been removed. Use https://github.com/ngokevin/aframe-template-component in the meantime. (#883)
+- `<a-camera>` no longer creates a cursor on its own. Do `<a-camera><a-cursor></a-cursor></a-camera>` instead. (#883)
+- Default geometry `depth`, `height`, `width` property values changed from 2 to 1. (#1245)
+- Default color of primitive elements such as `<a-box>` changed to `#FFF` to not interfere with textures. (#1245)
 
 ### Deprecations
 
-* `loader` component deprecated in favor of `collada-model` and `obj-model`. (#913)
-* `<a-model>` deprecated in favor of `<a-collada-model>` and `<a-obj-model>`. (#883)
-* `<a-EVENTNAME>` elements such as `<a-mouseenter>` deprecated in favor of `<a-event name="EVENTNAME">` (unstable). (#883)
-* `<a-cube>` deprecated in favor of `<a-box>`. (#883)
+- `loader` component deprecated in favor of `collada-model` and `obj-model`. (#913)
+- `<a-model>` deprecated in favor of `<a-collada-model>` and `<a-obj-model>`. (#883)
+- `<a-EVENTNAME>` elements such as `<a-mouseenter>` deprecated in favor of `<a-event name="EVENTNAME">` (unstable). (#883)
```

Comment:
you know diego and i have some qualms about it, and it's unreviewed. i'm not certain what is going to happen later with this API so i just want to give a heads up about that.